### PR TITLE
pythonPackages.awkward: 1.0.2 -> 1.1.2

### PR DESCRIPTION
**Pull request by veprbl**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 113953` [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>python39Packages.awkward</li>
  </ul>
</details>
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>python37Packages.awkward</li>
    <li>python38Packages.awkward</li>
  </ul>
</details>

**Review by davidak — CHANGES_REQUESTED** [review]
Change LGTM

Builds, except python39Packages.awkward

**Comment by davidak** [comment]
@veprbl can you have a look? It seem to build in their CI with python 3.9.

```
builder for '/nix/store/yiafg1glbw6mn5f3m99hsjwljmf76jgw-python3.9-awkward-1.1.2.drv' failed with exit code 1; last 10 log lines:
  E               def f1(array):
  E                   return 0 in array, 3 in array, 4 in array
  E                   ^
  
  /nix/store/nvafxrdxn8zmicmhxqfsc4nm8icl6g99-python3.9-numba-0.51.2/lib/python3.9/site-packages/numba/core/utils.py:80: UnsupportedError
  =========================== short test summary info ============================
  FAILED tests/test_0290-bug-fixes-for-hats.py::test_unmasked - numba.core.erro...
  FAILED tests/test_0290-bug-fixes-for-hats.py::test_indexedoption - numba.core...
  FAILED tests/test_0494-numba-array-contains.py::test_numba - numba.core.error...
  ================== 3 failed, 652 passed, 18 skipped in 28.29s ==================
```

**Comment by veprbl** [comment]
@davidak python39 is not officially supported in any currently existing release of numba

**Comment by davidak** [comment]
can you disable it then for now?

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113953` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python39Packages.awkward: <a href="https://termbin.com/2gxg">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/807bd596b847cb758f0f0d42351239c1/raw/index.html">pretty log </a></li>
  </ul>
</details>
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.awkward</li>
<li>python38Packages.awkward</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>

  <summary>1 package failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python39Packages.awkward: <a href="https://termbin.com/ym55o">plain log</a>  |  <a href="https://htmlpreview.github.io/?https://gist.githubusercontent.com/SuperSandroBot/ded1fd8a7ef086c621ced278fa7971c5/raw/index.html">pretty log </a></li>
  </ul>
</details>
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.awkward</li>
<li>python38Packages.awkward</li>
</ul>
</details>

**Comment by veprbl** [comment]
> can you disable it then for now?

Done in e714a4c3760830d94b7433498871c874fd91a828

File: pkgs/development/python-modules/awkward/default.nix [hunk 1]
```diff
@@ -25,6 +25,7 @@ buildPythonPackage rec {
 
   checkInputs = [ pytestCheckHook numba ];
   dontUseSetuptoolsCheck = true;
+  pytestFlagsArray = [ "--ignore=tests-cuda/" ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/awkward/default.nix:28]:
```suggestion
  disabledTestPaths = [ "tests-cuda" ];
```

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package marked as broken and skipped:
</summary>
<ul>
<li>python39Packages.awkward</li>
</ul>
</details>
<details>
<summary>
2 packages built:
</summary>
<ul>
<li>python37Packages.awkward</li>
<li>python38Packages.awkward</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 113953` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>python38Packages.awkward</li>
</ul>
</details>

**Pull request merged**